library/bsize.py

```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/python
+# Copyright (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
- should the modules be (c) Ansible Project? And why 2017?
- should they be GPL, when the rest of the role is MIT?

@timflannagan1 @dwlehman did you write those modules? What do you think?